### DISCUSSION TITLE
Integrate a community finder into Lemmy

### DISCUSSION THREAD

**Discussion by antoninadert** [aeharding/voyager#742] [Ideas]
If Voyager is the name, it should be easy to reach for new destinations. 

Today it is hard to explore on the lemmyverse. Many new users are onboarding and don't know where to look for new communities.
Solutions like [lemmyverse](https://lemmyverse.net/communities) exist and are good. 

My suggestion is to improve the Search page to be more helpful when it comes to finding new communities (trending communities is not sufficient).

I did a quick image edit to give an idea of what it would look like to integrate lemmyverse information into voyager.

Of course, we need to make it a better design before it goes into Voyager.

Here is the idea : 

<img width="762" alt="image" src="https://github.com/aeharding/voyager/assets/8929314/0f567e6f-9c22-4f4b-a16d-ed7444eae565">

**Comment by aeharding** [comment]
Hi there!

We could potentially add a "top communities" section. Because this is a bit open ended, I'm converting to a discussion.
